ratis-server/src/main/java/org/apache/ratis/server/impl/LeaderStateImpl.java

```diff
@@ -951,9 +951,11 @@ private void yieldLeaderToHigherPriorityPeer() {
     for (LogAppender logAppender : senders.getSenders()) {
       final FollowerInfo followerInfo = logAppender.getFollower();
       final RaftPeerId followerID = followerInfo.getPeer().getId();
-      final RaftPeer follower = conf.getPeer(followerID);
+      final RaftPeer follower = conf.getPeer(followerID, RaftPeerRole.FOLLOWER);
```

Comment:
There is no need to increase this variable, the default is to take FOLLOWER

Reply:
But when designing this method parameter, we thought that the variable will only be specified when the Listener is obtained. Also, this kind of call is a lot in ratis, see below, so I think we should remove this change

![image](https://user-images.githubusercontent.com/26144703/178487125-a4b88819-9458-4095-aba7-820357ca78f2.png)